terraform/stages/gke/main.tf

```diff
@@ -31,23 +31,22 @@ module "gke" {
   region         = var.region
   min_node_count = 1
   max_node_count = 10
-  machine_type   = "n1-standard-8"
+  machine_type   = "n1-standard-4"
 
   # This service account will be created in both GCP and GKE, and will be
   # used for workload federation in all microservices.
   # See microservices/sample_service/kustomize/base/deployment.yaml for example.
   service_account_name = "gke-sa"
 
   # See latest stable version at https://cloud.google.com/kubernetes-engine/docs/release-notes-stable
-  kubernetes_version = "1.23.14-gke.1800"
+  kubernetes_version = "1.23.16-gke.1400"
 }
 
 module "ingress" {
   depends_on = [module.gke]
 
-  source            = "../../modules/ingress"
+  source            = "../../modules/ingress_gce"
```

Comment:
Can you revert this back to using the original nginx-ingress until we have tested it throughouly?

Reply:
Done